recipes/json-spec/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "json-spec" %}
+{% set version = "0.10.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/json-spec-{{ version }}.tar.gz
+  sha256: 0b32fb0508d47115eccb4daa14731f101f1263fb57a8cb13ca49c960b05eb719
+
+build:
+  number: 0
+  entry_points:
+    - json = jsonspec.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - pathlib  # [py==27]
```

Comment:
Do you want to still support python 2.7?
just a heads up, ``grayskull`` has an option now which is called ``--strict-conda-forge``.

```
grayskull pypi --strict-conda-forge json-spec
```
Which will strip python 2.7 stuffs and will do some more magic, it is not perfect, but it helps
:smile:

Reply:
If you do not want to support python 2.7 we can remove it then
```suggestion
```
and add it as ``noarch: python``